app/presenters/autocomplete_items_presenter.rb

```diff
@@ -23,15 +26,22 @@ def pages_with_autocomplete_component
   end
 
   def messages
-    @messages ||=
-      pages_with_autocomplete_component.each_with_object([]) do |page, arry|
-        page.components.each do |component|
-          next unless component.uuid.in?(component_uuids_without_items)
-
-          arry.push(
-            { component_title: component.humanised_title, page_uuid: page.uuid }
-          )
-        end
+    pages_with_autocomplete_component.each_with_object([]) do |page, arry|
+      page.components.each do |component|
+        next unless component.uuid.in?(component_uuids_without_items)
+
+        link = govuk_link_to(component.humanised_title, Rails.application.routes.url_helpers.edit_page_path(service.service_id, page.uuid))
+
+        description = I18n.t("publish.autocomplete_items.#{deployment_environment}.message", title: link)
```

Comment:
```suggestion
        msg = I18n.t(
          "publish.autocomplete_items.#{deployment_environment}.message",
          title: link
        ).html_safe
```
Not a biggie, but we could chain the `html_safe` in the variable assignment and rename the variable, then we can delete the `msg = description.html_safe` line.